### DISCUSSION TITLE
Which static site generator are you using?

### DISCUSSION THREAD

**Discussion by estruyf** [estruyf/vscode-front-matter#68] [General]
I would like to know which static site generator you are using to see how the extension can future enhance the management experience. Feel free to share what you are using, and which integration you would like to see.

**Comment by Fasani** [comment]
Gatsby

**Reply by estruyf** [reply]
@Fasani  if you are using Front Matter, and want to be on our showcase page (https://frontmatter.codes/showcase). Feel free to submit your site: [submit showcase](https://github.com/estruyf/vscode-front-matter/issues/new?assignees=&labels=&template=showcase.md&title=Showcase%3A+).

**Reply by estruyf** [reply]
Are you using any shortcodes/snippets/custom scripts during your article creation/managing process?

**Reply by Fasani** [reply]
Currently, I just run pure markdown. I have been trying to future proof my content. Part of me is considering a sandboxed / preview of running code etc but I have not yet decided on an approach but I would like to bring some more interactive content to some pages.

**Comment by Programazing** [comment]
Hugo

**Comment by zivbk1** [comment]
Hugo and SvelteKit

**Reply by estruyf** [reply]
@zivbk1 for SvelteKit, is there an SSG you are using? Or did you create something yourself?

**Reply by zivbk1** [reply]
@estruyf SvelteKit can render a static site, see https://kit.svelte.dev/docs#adapters 
Which can be deployed on systems like Netlify, Vercel, Cloudflare Pages, etc.

It is the most similar to Hugo to me, when compared to the other Vue and React based frameworks.
Instead of Go Template language for the layouts, you use HTML, CSS, JS, and some nice features of Svelte and SvelteKit to add some extra functionality.

**Comment by Maximization** [comment]
Eleventy (11ty)

**Reply by estruyf** [reply]
Thanks @Maximization great to see how it works with other SSGs

**Comment by wemasoe** [comment]
jekyll and different themes

**Reply by estruyf** [reply]
Thanks! Great to see that many are using different SSGs!

**Comment by git-no** [comment]
SvelteKit at Vercel

**Comment by rufus42** [comment]
Hugo

**Comment by davidsneighbour** [comment]
Hugo, Svelte, Gatsby, Eleventy, Jekyll.

**Comment by lorezyra** [comment]
Hexo 6

**Comment by lolbat** [comment]
Hugo
